docs/user_guides/projects/jobs/schedule_job.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Documentation on how to schedule a job on Hopsworks.
+---
+
+# How To Schedule a Job 
+
+## Introduction
+
+Hopsworks jobs can be scheduled to run at regular intervals using the scheduling function provided by Hopsworks. Each job can be configured to have a single schedule.
+
+Schedules can be defined using the drop down menus in the UI or a [cron](https://en.wikipedia.org/wiki/Cron) expression. 
```

Comment:
Probably not relevant enough for the user to mention it here, but we use Quartz as [cron type](https://github.com/logicalclocks/hopsworks-ee/pull/1498/files#diff-9858d23bcd82fbbbb0a402ebaf5a8602e2f8c00bc4005a44edb8701766446e76R47)?

Reply:
Good catch let me add it